quilt/tools/store.py

```diff
@@ -105,6 +109,17 @@ def save_df(self, df, name, path, ext, target):
         """
         raise NotImplementedError()
 
+    def save_file(self, srcfile, name, path, target):
+        """
+        Save a (raw) file to the store.
+        """
+        self._find_path_write()
+        filehash = digest_file(srcfile)
+        fullname = name.lstrip('/').replace('/', '.')
```

Comment:
The store doesn't keep a running path or pointer into the JSON tree so we need to specify the full path when saving each dataframe.